admin-dev/themes/default/public/theme.rtlfix

```diff
@@ -1,129 +1,129 @@
 @font-face {
-  font-family: Vazir;
-  src: url('https://res.cloudinary.com/ipresta/raw/upload/v1522045380/vazir_v18.0.0/Vazir.eot');
-  src: url('https://res.cloudinary.com/ipresta/raw/upload/v1522045380/vazir_v18.0.0/Vazir.eot?#iefix') format('embedded-opentype'),
-       url('https://res.cloudinary.com/ipresta/raw/upload/v1522045380/vazir_v18.0.0/Vazir.woff') format('woff'),
-       url('https://res.cloudinary.com/ipresta/raw/upload/v1522044764/vazir_v18.0.0/Vazir.ttf') format('truetype');
-  font-weight: normal;
+    font-family: Vazir;
```

Comment:
Please do not change the whole indentation :/

Reply:
The reason is that I copied all the codes into my PHPStorm and it changed the whole indentations. Is it necessary to undo them?

Reply:
Yes, we use 2 spaces for indentation in css for this project.
It also helps to see what is the exact change.